### ISSUE TITLE
Getting error message trying to use examples from Readme

### ISSUE THREAD

**Issue by konung** [alexrudall/ruby-openai#139]
**Describe the bug**
Hi . 
Getting error messages from OpenAI endpoints trying to use  simple examples from Readme verbatum

**To Reproduce**
```ruby
client = OpenAI::Client.new(access_token: API_KEY_NG_PERSONAL)
response = client.images.generate(parameters: { prompt: "A baby sea otter cooking pasta wearing a hat of some sort" })

=> {"error"=>
       {"code"=>nil,
         "message"=>"'{\"prompt\":\"A baby sea otter cooking pasta wearing a hat of some sort\"}' is not of type 'object'",      
         "param"=>nil, 
         "type"=>"invalid_request_error"}}`
```

**Expected behavior**
Supposed to get a proper response as described in Readme :) 

**Screenshots**

**Desktop (please complete the following information):**
 - macosx - Ventura
 - Ruby 3.1.2 & Ruby 3.2.0
 
 **Extra Details**
 
 If I run:
 
 ```ruby
 client.embeddings(
        parameters: {
            model: "babbage-similarity",
            input: "The food was delicious and the waiter..."
        }
    )
```
    
works as expected. So I assume my setup is ok, the issues is with `client.images.generate` request itself?

**Comment by alexrudall** [comment]
Hi @konung - thanks for raising this, it was indeed a bug! Fix released in [v3.0.2](https://rubygems.org/gems/ruby-openai/versions/3.0.2)

**Comment by konung** [comment]
Confirmed: it works now :D 

Thank you!
